docs/state.md

```diff
@@ -445,6 +445,7 @@ WindowStore
   ledgerInfo: {
     creating: boolean,          // wallet is being created
     created: boolean,           // wallet is created
+    reconcilePeriod: number,    // duration of each reconciliation period in days
```

Comment:
Totally a nitpick- would it be better to say frequency? like the reconcileFrequency would be every 30 days.  When I see period or duration, it's not immediately clear